build/node.json

```diff
@@ -4,7 +4,8 @@
   "metadata": {
     "name": "fb4ebb70-2783-42b8-b3ef-63e2fd6d242e",
     "labels": {
-      "name": "edge-node"
+      "name": "edge-node",
+      "node-role.kubernetes.io/edge":"edge"
```

Comment:
"node-role.kubernetes.io/edge":"edge" -> "node-role.kubernetes.io/edge":""

Since we are not using the value of the label , it's ok to keep it empty